openCVLibrary331/src/main/java/org/opencv/core/Core.java

```diff
@@ -253,6 +253,20 @@ public static String getBuildInformation()
 
 
     //
+    // C++:  String getIppVersion()
+    //
```

Comment:
all these changes in opencv, did you make them or are they just a new version?

Reply:
Just a new version. 3.3.0 -> 3.3.1